pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
+name = "grimoirelab-bestiary"
+version = "0.1.0-dev"
+description = "A tool to visually manage software development ecosystems description."
+authors = ["GrimoireLab Developers"]
+license = "GPL-3.0+"
+
+[tool.poetry.dependencies]
+python = "^3.6"
+django = "3.1"
+grimoirelab_toolkit = ">=0.1.8"
+mysqlclient = "2.0.3"
+python-dateutil = ">=2.8.0"
+graphene-django = "^2"
+django-mysql = ">=3.2.0"
```

Comment:
Just a small nit, can we have all the dependencies written as `django-mysql = "^3.2.0"`.
While both the cases work, I was just thinking about uniformity. :sweat_smile:

Reply:
@vchrombie I agree with you.

Reply:
In any case, `django-mysql` is not needed anymore.